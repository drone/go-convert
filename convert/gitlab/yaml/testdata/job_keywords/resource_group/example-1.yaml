# https://docs.gitlab.com/ee/ci/yaml/#resource_group

deploy-to-production:
  script: deploy
  resource_group: production