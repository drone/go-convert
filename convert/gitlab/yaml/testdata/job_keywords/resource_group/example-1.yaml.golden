deploy-to-production:
    resource_group: production
    script:
        - deploy
